doc/input-output-reference/src/overview/group-advanced-surface-concepts.tex

```diff
@@ -2635,6 +2635,57 @@ \subsubsection{Field: Outdoor Air Node Name}\label{field-zone-outdoor-air-node-n
     OutdoorAirNode:0001;          !- Outdoor Air Node Name    
 \end{lstlisting}
 
+
+\subsection{SurfaceProperty:HeatBalanceSourceTerm}\label{surfacePropertyHeatBalanceSourceTerm}
+
+The object allows additional heat source term of the surface boundary condition in the surface heat balance calculation for Advanced/Research Usage. The object specifies an exterior surface with the additional heat source term described by a schedule of heat rates (can be positive or negative values) in W/m^2. The heat rates are pre-calculated outside EnergyPlus.
```

Comment:
Can this only be applied to exterior surfaces with an "outdoors" boundary condition?

Reply:
Not really. We modified the docs.